site/app/templates/submission/RainbowGrades.twig

```diff
@@ -1,16 +1,39 @@
 <div class="content">
-    <h1 class="label">
-    {% if extra_label is defined %}
-        Rainbow Grades {{extra_label}}
-    {% else %}
-        Rainbow Grades
-    {% endif %}
-    </h1>
+    <header>
+        <h1 class="label">
+            {% if extra_label is defined %}
+                Rainbow Grades {{extra_label}}
+            {% else %}
+                Rainbow Grades
+            {% endif %}
+        </h1>
+        <div>
+            {% if show_summary and extra_label is defined and is_gradebook is defined and is_gradebook %}
+                <div class="btn-wrapper" style="margin:auto;">
+                    <button
```

Comment:
you are having a lot of the styling defined in the twig file, perhaps you should try to move it to a css file?